domain/src/main/kotlin/no/nav/su/se/bakover/domain/Sak.kt

```diff
@@ -11,6 +11,7 @@ import no.nav.su.se.bakover.domain.søknadsbehandling.Søknadsbehandling
 import java.time.Clock
 import java.util.UUID
 
+// TODO ai 04.04.2021: Bytt till String og husk database endring
```

Comment:
Når/hvorfor bestemte vi oss for å bytte til string internt?

Reply:
Kan vel heller endre TODOen til å håndtere en eller annen form for validering av saksnummeret.. P.t. så vet vi at det er numerisk (som jo dekkes av en long) og minst 4 siffer (som ikke så godt dekkes av en long).

Reply:
Enig. privat ctor og fun create og tryCreate :+1: